Plugins/SafeDIGenerator/SafeDIGenerateDependencyTree.swift

```diff
@@ -46,32 +46,43 @@ struct SafeDIGenerateDependencyTree: BuildToolPlugin {
                     .sourceFiles(withSuffix: ".swift")
                     .map(\.url)
             }
-        let inputSourcesFilePath = context.pluginWorkDirectoryURL.appending(path: "InputSwiftFiles.csv").path()
-        try Data(
-            (targetSwiftFiles.map { $0.path(percentEncoded: false) } + dependenciesSourceFiles.map { $0.path(percentEncoded: false) })
-                .joined(separator: ",")
-                .utf8
-        )
-        .write(toPath: inputSourcesFilePath)
+        let inputSourcesFile = context.pluginWorkDirectoryURL.appending(path: "InputSwiftFiles.csv")
+        try (targetSwiftFiles.map { $0.path(percentEncoded: false) } + dependenciesSourceFiles.map { $0.path(percentEncoded: false) })
+            .joined(separator: ",")
+            .write(
+                to: inputSourcesFile,
+                atomically: true,
+                encoding: .utf8
+            )
+
+        let includeCSV = context.safediFolder.appending(components: "configuration", "include.csv")
+        let includeArguments: [String] = if FileManager.default.fileExists(atPath: includeCSV.path()) {
+            [
+                "--include-file-path",
+                includeCSV.path(),
+            ]
+        } else {
+            []
+        }
+        let additionalImportedModulesCSV = context.safediFolder.appending(components: "configuration", "additionalImportedModules.csv")
+        let additionalImportedModulesArguments: [String] = if FileManager.default.fileExists(atPath: additionalImportedModulesCSV.path()) {
+            [
+                "--additional-imported-modules-file-path",
+                additionalImportedModulesCSV.path(),
+            ]
+        } else {
+            []
+        }
+
         let arguments = [
-            inputSourcesFilePath,
+            inputSourcesFile.path(),
             "--dependency-tree-output",
             outputSwiftFile.path(),
-        ]
+        ] + includeArguments + additionalImportedModulesArguments
 
         let downloadedToolLocation = context.downloadedToolLocation
         let safeDIVersion = context.safeDIVersion
-        if context.hasSafeDIFolder, let safeDIVersion, downloadedToolLocation == nil {
-            Diagnostics.error("""
-            \(context.safediFolder.path()) exists, but contains no SafeDITool binary for version \(safeDIVersion).
-
-            To install the release SafeDITool binary for version \(safeDIVersion), run:
-            \tswift package --package-path \(context.package.directoryURL.path()) --allow-network-connections all --allow-writing-to-package-directory safedi-release-install
-
-            To use a debug SafeDITool binary instead, remove previous installs by running:
-            \trm -rf \(context.safediFolder.path())
-            """)
```

Comment:
we used to error if you previously installed the release version and then changed versions. I'm deleting that code path because:

1. I'm not convinced it's super helpful. We still have the warnings
2. It's harder (not impossible) to tell whether the folder exists because we've downloaded a release binary in the past now that we're reusing this folder for configuration setting